runtime/Java/src/org/antlr/v4/runtime/misc/InterpreterDataReader.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2012-2017 The ANTLR Project. All rights reserved.
+ * Use of this file is governed by the BSD 3-clause license that
+ * can be found in the LICENSE.txt file in the project root.
+ */
+
+package org.antlr.v4.runtime.misc;
+
+import org.antlr.v4.runtime.Vocabulary;
+import org.antlr.v4.runtime.VocabularyImpl;
+import org.antlr.v4.runtime.atn.ATN;
+import org.antlr.v4.runtime.atn.ATNDeserializer;
+import org.antlr.v4.runtime.dfa.DFA;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.ArrayList;
+import java.io.BufferedReader;
+import java.io.FileReader;
+
+// A class to read plain text interpreter data produced by ANTLR.
+public class InterpreterDataReader {
```

Comment:
is not clear who uses this without using an IDE. does the ParserInterpreter know how to use this somehow?